### PR TITLE
Assorted Content Changes

### PR DESCRIPTION
**Pull request by quarterback**
This PR didn't push before the holiday. Made additional edits that weren't included in original PR.

### This pull request changes...
- #753 - language around key personnel percentages/FTE
- #749 - Activity summary language fixes
- #748 - Content context for 280 character limit in program activities
- #750 - Added update
- #747 - Program overview differentiation text
- #737 - FTE% help text

### This pull request is ready to merge when...
- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [ ] The change has been documented
  - [ ] Associated OpenAPI documentation has been updated

### This feature is done when...
- [ ] Product has approved the experience

### CODE REVIEW

File: web/src/i18n/locales/en.yaml [hunk 1]
```diff
@@ -142,13 +142,13 @@
         subheader: "Brief Activity Summary"
       detail:
         standard:
-          subheader: "Describe the activity and include relevant information."
+          subheader: "There will be opportunities to add aditional text in other sections."
```
Comment (NAretakis) [inline web/src/i18n/locales/en.yaml]: How about this teak? "This should very brief, one or two sentences describing the activity. There will be opportunities to add additional text in other sections"

File: web/src/i18n/locales/en.yaml [hunk 2]
```diff
@@ -48,12 +48,12 @@
...
     overview: 
       title: "Program Overview"
-      helpText: "Provide a high level summary of your state's program, history, and future vision."
+      helpText: "Provide a high level summary of your state's program, history, and future vision. This initial overview is intended to be brief, there will be opportunities to explain in more detail in the activity details section."
       reminder: "Indicate the federal fiscal years for which this APD is requesting funding."
       yearsCovered: "What federal fiscal year(s) does this APD cover?"
       labels: 
         desc: "Program Introduction"
-        helpText: "Describe the scope of what you intend to do. It is not meant to replicate the executive summary section. You will have an opportunity to explain in more detail later in this application."
+        helpText: "Describe the scope of what you intend to do. This summary will be used in other parts of this application. You will have an opportunity to explain in more detail in the activity details section."
```
Comment (NAretakis) [inline web/src/i18n/locales/en.yaml]: For some reason the helptext here is sitting above the Program Introduction section, as compared to HIE and MMIS where it sits below. Not sure if this is fixable here, or in another part of the app. Thoughts?

Comment (mgwalker) [reply web/src/i18n/locales/en.yaml]: The `Program Introduction` part doesn't have the same layout as the other two for some reason.  That'll need to be a dev fix, but super easy.  I'll add it on this branch so it'll go in at once.

Comment (jeromeleecms) [reply web/src/i18n/locales/en.yaml]: "This initial overview is intended to be brief, **there will be opportunities to explain in more detail in the activity details section.**"

This reads like a run-on sentence(?) how about (catching tonal changes where we can):

"Provide a high level summary of the state's program, history, and future vision. Keep it brief. Specific activities for the program will be explained in the program activities section."

File: web/src/i18n/locales/en.yaml [hunk 3]
```diff
@@ -172,18 +172,18 @@
...
       title: "Activity Overview"
       helpText: "Summarize the needs of the activity for which requested funding will be used (e.g. administrative, outreach, auditing, Public Health Registry Development, etc)"
-      reminder: "This should be very brief."
+      reminder: "The activity overview should be a brief summary of the activity needs."
       summary: 
         subheader: "Brief Activity Summary"
       detail:
         standard:
-          subheader: "Describe the activity and include relevant information."
+          subheader: "The activity summary should be brief, there will be additional opportunities to expand in other sections."
```
Comment (jeromeleecms) [inline web/src/i18n/locales/en.yaml]: This section below the brief activity summary doesn't have to be brief. This should be as long as needs to be in order to explain what they are doing. Are we missing a title for this text box?

image

File: web/src/i18n/locales/en.yaml [hunk 4]
```diff
@@ -172,18 +172,18 @@
...
       alternatives: 
         subheader: "Statement of alternative considerations and supporting justification"
-        helpText: "If no substantive changes have been made to this activity, include a statement referring to the Statement of Alternative Considerations contained in previous versions of the IAPD or in the SMHP. If substantive changes have been made, include a statement describing the alternatives your state considered when making the change."
+        helpText: "If no substantive changes have been made to this activity, include a statement referring to the Statement of Alternative Considerations contained in previous versions of the IAPD update or in the SMHP. If substantive changes have been made, include a statement describing the alternatives your state considered when making the change."
```
Comment (jeromeleecms) [inline web/src/i18n/locales/en.yaml]: For this line: "If substantive changes have been made, include a statement describing the alternatives your state considered when making the change."

Suggestion (feel free to revise - point is that also covers new activities): "If substantive changes have been made OR *this is a new activity*, include a statement describing what alternative considerations were made before selecting the strategy for implementation."

File: web/src/i18n/locales/en.yaml [hunk 5]
```diff
@@ -232,7 +232,7 @@
     statePersonnel: 
       title: "Personnel"
       subheader: "Outline the personnel resources for this section. This includes anyone funded as a part of this activity. This does not include personnel listed in the contract section."
-      helpText: "Personnel costs must include salaries and benefits. FTE is percentage attributable to the activity. Enter individuals only once."
+      helpText: "Personnel costs for this section includes the combined salary and benefits. FTE% is the percentage attributable to the activity. You only need to enter personnel once. Percentages are generated automatically, you only have to insert a number in each section by FFY."
```
Comment (jeromeleecms) [inline web/src/i18n/locales/en.yaml]: - Should include some language to indicate that Salary amount is the yearly cost of the individual.
- FTE% -> %FTE to match what's currently being displayed
- "Percentages are generated automatically" is a bit misleading. Maybe "Yearly costs are automatically calculated from the %FTE, provided." ?

File: web/src/i18n/locales/en.yaml [hunk 6]
```diff
@@ -308,7 +308,7 @@
...
         title: "Describe how you'll meet the Mitigation Strategy condition"
       keyPersonnel: 
-        header: "Key Personnel"
+        header: Personnel"
```
Comment (jeromeleecms) [inline web/src/i18n/locales/en.yaml]: Key Personnel is actually the correct term for this citation. Looks like the "Key (with quote) was removed.

**Comment by jeromeleecms** [comment]
@quarterback Made some comments in the text. Tried to identify where things were incorrect.

First time working on help text (I just realized that @NAretakis always took these!) - if you prefer a different way for me to put in comments/edits, please let me know.

File: web/src/i18n/locales/en.yaml (hunk 6 above)
Comment (quarterback) [reply web/src/i18n/locales/en.yaml]: Will fix.

File: web/src/i18n/locales/en.yaml (hunk 5 above)
Comment (quarterback) [reply web/src/i18n/locales/en.yaml]: ok

Comment (jeromeleecms) [reply web/src/i18n/locales/en.yaml]: @quarterback What did you think about bullet 3?

File: web/src/i18n/locales/en.yaml [hunk 7]
```diff
@@ -334,8 +334,10 @@
     helpText: "This itemized budget table has combined the costs for each activity into a single itemized table by federal fiscal year (FFY). The table displays total share, federal share, and state share. The second table also includes a proposed breakout of the budget by FFY quarter."
     summaryBudget: 
       title: Summary Budget Table
+      helpText: "Totals are calculated automatically."
     quarterlyBudget: 
       title: Federal Share by FFY Quarter
+      helpText: "Personnel costs for this section includes combined salary and benefits. Salary amount is the yearly cost of the individual personnel listed and personnel only need to be listed once. FTE% is the percentage attributable to the activity. Percentages are automatically calculated from the %FTE provided."
```
Comment (jeromeleecms) [inline web/src/i18n/locales/en.yaml]: @quarterback Was this in the wrong place? Looks like this should be at line 235 to address the previous comment?

Comment (quarterback) [reply web/src/i18n/locales/en.yaml]: oops

**Comment by brendansudol** [comment]
looks like all fixes are in -- let's do this :)

**Pull request merged**